parasoft-findings-teamcity-agent/src/main/resources/com/parasoft/findings/teamcity/agent/sa-pmd.xsl

```diff
@@ -110,6 +111,25 @@
         </xsl:element>
     </xsl:template>
 
+    <xsl:template match="ElDesc">
+        <xsl:text>&#xa;        - </xsl:text>
+        <xsl:call-template name="firstLevelList"/>
```

Comment:
i think it would be better for reading if put line 115 in to **firstLevelList** template.